examples/machine_translation/README.md

```diff
@@ -0,0 +1,9 @@
+# This is an example to create a machine translation dataset and train a translation model.
+
```

Comment:
Should we include a metric for the test/valid datasets with the trained model? See my comments about `bleu_score` below.

Reply:
That would be better. However, I may not be able to run a full blown training as my resource is quite limited. Do you have any suggestions?

Reply:
Never mind. I will find time and work on it this half. Then, I can update this. Just to make sure that you set up the model/training correctly by checking the learning curve.

Reply:
Got it. Sorry for the trouble 🙏

Reply:
borrowed a resource to run 10 epochs, already putting the result on README. wdyt?

Reply:
It might be a little bit too long. Should we just include the final test result?

Reply:
Instead of removing the whole training metrics from the docs I trim it so that I only include the first and the last training output to give users some idea on the loss value while running the example